html/elements/script.json

```diff
@@ -603,6 +603,43 @@
                 "standard_track": true,
                 "deprecated": false
               }
+            },
+            "integrity": {
+              "__compat": {
+                "description": "<code>integrity</code>",
+                "spec_url": "https://github.com/whatwg/html/pull/10269",
+                "tags": [
+                  "web-features:import-map-integrity"
+                ],
+                "support": {
+                  "chrome": {
+                    "version_added": "127"
```

Comment:
This is speculative, as the feature wasn't yet enabled by default.

Reply:
If it isn't enabled, you'll need to put the necessary flag info to enable it; see https://github.com/mdn/browser-compat-data/blob/main/schemas/compat-data-schema.md#flags

Reply:
CL landed